docs/customizing.rst

```diff
@@ -0,0 +1,198 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2020 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Customizing
+===========
+
+
+how to inject/override the Invenio config (url, facets, the DOM id where to render the app): via parametrize or remove the Jinja macro and inject JSON config
+
+Configuring the settings of your Search Application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A helper is provided to generate the configuration needed for the
+React-Searchkit application.
+This configuration contains the following information:
+
+`aggs`: a list of aggregations that will add accordingly facets to your search.
```

Comment:
```suggestion
`aggs`: A list of aggregations that will accordingly add facets to your search.
```

Reply:
```suggestion
`aggs`: a list of aggregations that will add facets to your search accordingly.
```
Maybe this would be even more appropriate.